backfiller/backfiller/main.py

```diff
@@ -0,0 +1,200 @@
+
+# this is a prototype of the backfiller
+# lots about web services and the like I don't know
+# needs logging, exception handling and the like
+# also proper doc strings
+
+# when starting the backfiller and every few hours, backfill everything
+# more frequently, backfill the last couple hours
+# (last three hour directories so always at least two hours).
+
+import os
+import time
+import datetime
+import errno
+import uuid
+
+import requests
+
+import common
+
+HOUR_FMT = '%Y-%m-%dT%H'
+TIMEOUT = 5
```

Comment:
should add a comment saying what the timeout is for eg. "timeout on all requests to remote servers"